low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view

```diff
@@ -1023,6 +1026,7 @@ class SSPLHealthView(object):
             'disks': {}
         }
         cortx_build_version = 'NA'
+        # TODO: Remove salt dependency
         cortx_build_cmd = f"sudo salt-call pillar.get release:target_build --output=newline_values_only"
```

Comment:
Resource healthview mainly used in integrated env, stitched by provisioner across nodes and used by CSM for holistic resource view across cluster. For single node component vm deployment task, this wont be complete anyways, so lets take it in coming days. Create a jira ticket for same though

Reply:
Created EOS-16668 and this will be started during integration stage.